source/reference/program/mongostat.txt

```diff
@@ -508,3 +510,37 @@ members of the cluster:
 .. code-block:: sh
 
    mongostat --discover
+
+.. _example-mongostat-interactive:
+
+View Statistics in an Interactive Interface
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 3.4
+
+Use the :option:`--interactive <mongostat --interactive>` option to
+view statistics in a non-scrolling `ncurses<https://www.gnu.org/software/ncurses/>`_-style
+interactive output. :option:`--interactive` lets you highlight specific
```

Comment:
Nit: Not a huge fan of starting sentences with methods/options/whatevers that start with lowercase or symbols. Can we say "The :option:`--interactive` option lets you..." ?